packages/11ty/content/_assets/javascript/application/canvas-panel.js

```diff
@@ -10,21 +10,31 @@ window.onload = function () {
     if (!figure) return
     const annotationInputs = figure.querySelectorAll('.annotations-ui .annotations-ui__input')
     for (const annotationInput of annotationInputs) {
-      annotationInput.addEventListener('click', (event) => {
-        const id = event.target.getAttribute('id')
-        const type = annotationInput.getAttribute('data-annotation-type')
-        switch (type) {
-          case 'choice':
+      const id = annotationInput.getAttribute('id')
+      const type = annotationInput.getAttribute('data-annotation-type')
+      switch (type) {
+        case 'choice':
+          annotationInput.addEventListener('click', (event) => {
             canvasPanel.makeChoice(id)
-            break
-          case 'annotation':
-            // @todo select annotation
-            break
-          default:
-            console.warn(`Invalid annotation type ${type}`)
-            break
-        }
-      })
+          })
+          break
+        case 'annotation':
+          const toggleAnnotation = (input) => {
+            if (input.checked) {
+              canvasPanel.applyStyles(id, { opacity: 1 })
+            } else {
+              canvasPanel.applyStyles(id, { opacity: 0 })
+            }
+          }
```

Comment:
The double negative `!!` is an effective toggle and there are a number (intended) of ways to write converting the boolean `checked` value to an integer.
```suggestion
          const toggleAnnotation = ({ checked }) => {
            canvasPanel.applyStyles(id, { opacity: Number(!!checked) })
          }
```

Reply:
Alternately, using a bitwise operator
```
canvasPanel.applyStyles(id, { opacity: checked & 1 })
```
or
```
canvasPanel.applyStyles(id, { opacity: checked | 0 })
```